src/miral/regenerate-miral-symbols-map.py

```diff
@@ -555,11 +555,11 @@ def _report(publish, symbol):
   local: *;
 };
 
-MIRAL_4.1 {
+MIRAL_4.2 {
 global:'''
 
 END_NEW_STANZA = '''
-} MIRAL_4.0;
+} MIRAL_4.1;
```

Comment:
We need to clear these symbols from the script and let it fill in the current stanza. See 8e3130d7f30204b6d2545694218dab935f68e8a2 for an example